django_celery_results/urls.py

```diff
@@ -31,6 +31,7 @@ def to_url(self, value):
 register_converter(TaskPatternConverter, 'task_pattern')
 
 urlpatterns = [
+    # Old
```

Comment:
If you want normal deprecation I recommend to do two things:

1. Extract old `urlpatterns` to separate part and place it under `if`, with some setting, that is by default is `on`. This will allow to switch old urls off, and use new as proposed in #214
2. Raise deprecation warning if if is enabled. This will be raised on systems initialization and every correct CI/CD will track it. As well as developer if he did not switch off deprecation warnings.

Reply:
Ah you beat me to it :)